WordPress/src/test/java/org/wordpress/android/ui/reader/utils/FeaturedImageUtilsTest.kt

```diff
@@ -0,0 +1,89 @@
+package org.wordpress.android.ui.reader.utils
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.models.ReaderPost
+
+@RunWith(MockitoJUnitRunner::class)
+class FeaturedImageUtilsTest {
+    private val featuredImageUtils = FeaturedImageUtils()
+    @Test
+    fun `shows exactly the same featured image`() {
```

Comment:
Not fully sure to get the naming logic of these test cases. I would expect something like:

```
where asserThat(result).isTrue() -> means that the name should be "does not show..."
where asserThat(result).isFalse() -> means that the name should be "shows..."
```
Am I getting it wrong?

Reply:
oh you're absolutely right 🤦‍♂. It should be something like `does not show the same image twice`. Good point!

Reply:
Naming look great thanks! And 💯 for using TDD 😎!